CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
+    * Reconfigured the context menu for many items, especially the container app item [#518](https://github.com/microsoft/vscode-azurecontainerapps/pull/518)
+
+* New family of commands: `Deploy Project from Workspace`, `Create Container App from Workspace`, `Deploy Workspace to Container App` [#425](https://github.com/microsoft/vscode-azurecontainerapps/issues/425)
+    * Todo: Link to README?
+    * Creating Azure Container Registries [#435](https://github.com/microsoft/vscode-azurecontainerapps/pull/435)
+    * Dockerfile port smart detection for ingress configuration [#449](https://github.com/microsoft/vscode-azurecontainerapps/pull/449)
+    * Improve environment variable file smart detection logic [#450](https://github.com/microsoft/vscode-azurecontainerapps/pull/450)
+    * Save and re-deploy to Azure resources through workspace settings [#454](https://github.com/microsoft/vscode-azurecontainerapps/pull/454)
+    * Leverage a new form of activity log support for displaying multiple activities in a single command [#464](https://github.com/microsoft/vscode-azurecontainerapps/pull/464)
+    * Expand command entry-points to include container apps environment and container app items [#482](https://github.com/microsoft/vscode-azurecontainerapps/pull/482)
+
+* New revision draft editing mode for bundling deployment changes [#311](https://github.com/microsoft/vscode-azurecontainerapps/issues/311)
+    * Todo: Link to README?
+    * `Edit Container App (Advanced)` and `Discard Draft` in single revision mode [#405](https://github.com/microsoft/vscode-azurecontainerapps/pull/405)
+    * `Create Draft` and `Edit Draft (Advanced)` in multiple revision mode [#413](https://github.com/microsoft/vscode-azurecontainerapps/pull/413)
+    * `Deploy Draft` support (all modes) [#414](https://github.com/microsoft/vscode-azurecontainerapps/pull/414)
+    * Ability for container app template commands to hook into the new revision draft update system [#423](https://github.com/microsoft/vscode-azurecontainerapps/pull/423)
+    * Display a user setting controlled deployment pop-up when executing revision draft commands [#478](https://github.com/microsoft/vscode-azurecontainerapps/pull/478)
+
+* Log streaming support [#350](https://github.com/microsoft/vscode-azurecontainerapps/pull/350)
+* Deploying to a container app via a connected GitHub repository [#353](https://github.com/microsoft/vscode-azurecontainerapps/issues/313)
+* Tree view and CRUD support for container app secrets
+* Delete existing scale rules [#461](https://github.com/microsoft/vscode-azurecontainerapps/pull/461)
+* Improve activity log support for the majority of commands
+
+### Changed
+* Use smart detection to suggest values when deploying from a container registry [#305](https://github.com/microsoft/vscode-azurecontainerapps/pull/305), [#331](https://github.com/microsoft/vscode-azurecontainerapps/pull/331)
+* Revision draft commands:
```

Comment:
This feels like it belongs up in the portion that talks about bundling the draft changes

Reply:
Yeah, I think only reason it was in `changes` rather than `added` was because they were existing commands that were changed to become revision draft based

Reply:
I think it makes sense to have it in changes per Matt's reason.